src/components/screens/RecentlyUsedApps.tsx

```diff
@@ -0,0 +1,47 @@
+import useRecentConnection from "@/hooks/useRecentConnection";
+import AppCard from "../AppCard";
+import { Text } from "@ledgerhq/react-ui";
+import { t } from "i18next";
+import styled from "styled-components";
+
+const GridContainer = styled.div`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  gap: 10px;
+  justify-items: center;
+  overflow-y: scroll;
+`;
+
+const StyledDiv = styled.div`
+  background: ${({ theme }) => theme.colors.opacityDefault.c05};
+  min-width: 250px;
+  width: 250px;
+  border-radius: 5px;
+  cursor: pointer;
+`;
+
+export default function RecentlyUsedApps() {
+  const { lastConnectionApps } = useRecentConnection();
```

Comment:
We can use the `useAtomValue` directly
```suggestion
  const lastConnectionApps = useAtomValue(sortedRecentConnectionApps);
```